contracts/LiquidationsPeripheral.vy

```diff
@@ -930,3 +1000,11 @@ def adminLiquidation(_principal: uint256, _interestAmount: uint256, _liquidation
         _erc20TokenContract,
         "BACKSTOP_PERIOD_ADMIN"
     )
+
+@external
+def storeERC721CollateralToVault(_collateralAddress: address, _tokenId: uint256):
```

Comment:
We're implementing `onERC721Received` in `LiquidationPeripheral` so it can receive the collaterals from the vaults during NFTX liquidation.
The idea of this method is to have a safety valve in case an ERC721 gets transferred (eg by mistake) to this contract outside of the NFTX liquidation flow. 
Do you know if there's another way to solve it?

Reply:
> The idea of this method is to have a safety valve in case an ERC721 gets transferred (eg by mistake) to this contract outside of the NFTX liquidation flow.

Maybe there is but I'm not seeing a plausible scenario where this would happen. The only scenario would be if an NFT would be manually sent to this contract. We can leave the method for safety but I don't expect us to use it 🙂